src/utils/write-operations.ts

```diff
@@ -41,11 +41,15 @@ export const writeOperationsRepository = (
       Object.entries(api.deployments).forEach(([directoryName, deployments]) => {
         const subDeploymentBasePath = join(deploymentsBasePath, directoryName);
         mkdirSync(subDeploymentBasePath);
-        Object.entries(deployments).forEach(([deploymentName, deployment]) => {
+        Object.entries(deployments).forEach(([deploymentName, deploymentProvider]) => {
```

Comment:
Nit: what do you think of `cloudProvider` instead? I'm thinking that maybe the reader could think of either blockchain or cloud when reading it.